ee/tabby-db/src/web_documents.rs

```diff
@@ -58,6 +58,27 @@ impl DbConn {
         Ok(res.last_insert_rowid())
     }
 
+    pub async fn active_preset_web_document(&self, name: String, url: String) -> Result<i64> {
+        let res = query!(
+            "INSERT INTO web_documents(name, url, is_preset) VALUES (?,?,true);",
+            name,
+            url,
+        )
+        .execute(&self.pool)
+        .await?;
+
+        Ok(res.last_insert_rowid())
+    }
+
+    pub async fn deactivate_preset_web_document(&self, name: String) -> Result<i64> {
+        let res = query_scalar!("SELECT id FROM web_documents WHERE name = ?;", name)
+            .fetch_one(&self.pool)
+            .await?;
+
+        self.delete_web_document(res).await?;
+        Ok(res)
+    }
+
```

Comment:
```suggestion
```

As discussed: set an preset webdocument to active = `create`. set preset webdocument to inactive = `delete`.